src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -2052,6 +2052,25 @@ public void requestRefusedWhenAutomatedBlockExistsForPatron() {
       hasMessage(MAX_OUTSTANDING_FEE_FINE_BALANCE_MESSAGE))));
   }
 
+  @Test
+  public void cannotCreateRecallRequestForAgedToLostItem() {
+    final IndividualResource withdrawnItem = itemsFixture
+      .basedUponSmallAngryPlanet(ItemBuilder::agedToLost);
+
+    final Response response = requestsClient.attemptCreate(new RequestBuilder()
+      .open()
+      .recall()
+      .forItem(withdrawnItem)
+      .by(usersFixture.steve())
+      .fulfilToHoldShelf()
+      .withPickupServicePointId(servicePointsFixture.cd1().getId()));
+
+    assertThat(response, hasStatus(HTTP_UNPROCESSABLE_ENTITY));
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("Recall requests are not allowed for this patron and item combination"),
+      hasParameter("requestType", "Recall"))));
+  }
+
```

Comment:
Given that there is a unit test for this, what do you think to omitting this test (with the goal of moving more of the testing to unit style tests)?

Reply:
We may have a single API test to verify that it works well in integration.

Reply:
I completely agree (for the same reasons as above), please could the existing and new API tests be consolidated to a single test to demonstrate the use of white list